runtime/src/accounts_db.rs

```diff
@@ -88,6 +88,10 @@ pub const DEFAULT_NUM_DIRS: u32 = 4;
 // When calculating hashes, it is helpful to break the pubkeys found into bins based on the pubkey value.
 // More bins means smaller vectors to sort, copy, etc.
 pub const PUBKEY_BINS_FOR_CALCULATING_HASHES: usize = 65536;
+// # of passes should be a function of the total # of accounts that are active.
+// higher passes = slower total time, lower dynamic memory usage
+// lower passes = faster total time, higher dynamic memory usage
+// passes=2 cuts dynamic memory usage in approximately half.
 pub const NUM_SCAN_PASSES: usize = 2;
 pub const BINS_PER_PASS: usize = 32768; // PUBKEY_BINS_FOR_CALCULATING_HASHES / NUM_SCAN_PASSES
```

Comment:
And this constant could be calculated too, right?
```suggestion
pub const BINS_PER_PASS: usize = PUBKEY_BINS_FOR_CALCULATING_HASHES / NUM_SCAN_PASSES;
```